core/src/main/java/org/apache/calcite/sql/validate/SqlConformanceEnum.java

```diff
@@ -117,6 +121,7 @@ public enum SqlConformanceEnum implements SqlConformance {
     case LENIENT:
     case BIG_QUERY:
     case MYSQL_5:
+    case SPARK:
```

Comment:
Please edit the java doc for all the parent functions (in SqlConformance) of overridden functions which are changed. Add SPARK in the function description for which function will return true.

Reply:
Made changes according to the comment given above.